src/protocol/mod.rs

```diff
@@ -5,8 +5,12 @@ pub mod context;
 pub mod ipa;
 pub mod malicious;
 pub mod modulus_conversion;
+#[cfg(feature = "no-prss")]
```

Comment:
I am a bit surprised that you need to re-export those. I thought having `prss` module exported would be enough.

Reply:
Removing "pub" here but we would still need to re-export the traits/functions defined in no_prss or use_prss since they are not exposed in prss.rs anymore right? i.e. line 2-4 in prss.rs

Reply:
yep, removing `pub` is what I meant. It is still looks a bit upside down: `prss` module conditionally re-exports functions from parent submodules.

would it be cleaner if we move `prss` to its own folder inside the protocol? 

```
-> protocol
    -> prss
          -> mod.rs // conditionally re-exports either no-op PRSS or actual PRSS
          -> no_op.rs
          -> crypto.rs 
```

Reply:
Done